File: algotester0244.py
import sys
n = int(input())
liist = []
for i in range(n):
    liist.append(input())
if liist[0] == 'Out':
    print("Error")
    sys.exit()
In = liist.count('In')
Out = liist.count('Out')
if In == Out:
    print("Just a fantasy")
elif In > Out:
    print("Cunning elector")
elif In < Out:
    print("Error")

File: algotester0244.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
	int n, in = 0, out = 0;
	cin>>n;
	for(int i = 0; i < n; i++)
	{
		string a;
		cin>>a;
		if(a == "In")
			in++;
		else
			out++;	
	}
if(in > out)
	cout<<"Cunning elector";
else if (in == out)
	cout<<"Just a fantasy";
else
	cout<<"Error";
	return 0;
}